src/fosslight_util/oss_item.py

```diff
@@ -32,6 +35,9 @@ def __init__(self, value):
         self.exclude = ""
```

Comment:
Let's change default value of exclude to False

Reply:
OK, I changed it!